app/assets/stylesheets/companies/news_index.scss

```diff
@@ -0,0 +1,216 @@
+        max-width: 400px;
+        height: 260px;
+        overflow: hidden;
+  
+        .image-preview {
+          display: flex;
+          align-items: center;
+          position: relative;
+          overflow: hidden;
+
+          img {
+            width: 100%;
+            height: 100%;
+            object-fit: cover;
+            position: absolute;
+            top: 0;
+            left: 0;
+          }
+        }
+      }
+
+      .news-date-icon {
+        display: flex;
+
+        .news-icon {
+          margin: 5px 5px 5px 0px;
+        }
+
+        .news-date {
+          margin: 5px 0px 5px 5px;
+          font-weight: 500;
+          font-size: 14px;
+          line-height: 16px;
+        }
+      }
+
+      .news-title-block {
+        margin: 13px 0px 40px;
+
+        .news-title {
+          display: flex;
+          align-items: center;
+          font-weight: 500;
+          font-size: 20px;
+          line-height: 23px;
+          color: $black-color;
+
+          &:hover {
+            color: $base-color;
+          }
+        }
+      }
+    }
+  }
+
+  @media screen and (max-width: 1482px) {
+    .news-container {
+      justify-content: space-evenly;
+
+      .news-block {
+        max-width: 330px;
+      }
+    }
+  }
+
+  @media screen and (max-width: 1236px) {
+    .news-container {
+
+      .news-block {
+        max-width: 320px;
+      }
+    }
+  }
+
+  @media screen and (max-width: 1199px) {
+    .news-container {
+
+      .news-block {
+        max-width: 300px;
+      }
+    }
+  }
+
+  @media screen and (max-width: 1164px) {
+    .news-container {
+      justify-content: space-around;
+
+      .news-block {
+        min-width: 330px;
+      }
+    }
+  }
+
+  @media screen and (max-width: 999px) {
+    .news-container {
+      align-items: center;
+      height: 1000px;
+
+      .news-block {
+        min-width: 400px;
+      }
+    }
+  }
+
+  @media screen and (max-width: 900px) {
+    .news-container {
+      flex-flow: column nowrap;
+      justify-content: flex-start;
+
+      .news-block {
+        width: 100%;
+        overflow: visible;
+      }
+    }
+  }
+
+  @media screen and (max-width: 442px) {
+    .news-container {
+
+      .news-block {
+        max-width: 400px;
+        min-width: 280px;
+      }
+    }
+  }
+}
+
+.button-load-more {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  flex-wrap: nowrap;
+  justify-content: center;
+  margin-top: 40px;
+  margin-right:  20px;
+  margin-bottom: calc(50px + (70 - 50) * ((100vw - 320px) / (1920 - 320)));
+  margin-left:  20px;
+  padding: calc(10px + (22 - 10) * ((100vw - 320px) / (1920 - 320)));
+  width: calc(290px + (400 - 290) * ((100vw - 320px) / (1920 - 320)));
+  height: 60px;
+  background: $load_more_button;
+  border-radius: 20px;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 28px;
+  color: $white-color;
+  cursor: pointer;
+
+  &:hover {
+    background: $base-color;
+  }
+}
```

Comment:
minor comment: inconsistent indentation in the file

Reply:
corrected